docs/proposals/external-service-integration.md

```diff
@@ -0,0 +1,51 @@
+#External Service Integrations
+
+## Background
+https://github.com/kubernetes-incubator/service-catalog/issues/1455
+
+## Abstract
+
+This document aims to provide solutions to the following use cases: 
+
+1) A mobile enabled service has been provisioned via the service catalog into namespace A. It is intended for this to be a shared service. Think something like
+Authentication. 
+An MCP is provisioned in namespace B along with another mobile enabled service that is capable of integrating with the mobile service in namespace A. How do we support
+this kind of integration?
+
+
+2) A user or org has an existing service such as Keycloak that exists entirely separately to OpenShift, but is made accessible. It is the intention that this would be a
+shared service. A developer then creates a mobile project with MCP provisioned on an OpenShift instance, in what circumstances can we provide integration and what form
+should that integration take?
+
+
+## Cross Namespace integrations
+
+We are leveraging the machinery provided by the service catalog to power our integrations. This document assumes some level of familiarity with ServiceBindings and ServiceInstances and the ansible service broker.
```

Comment:
Could we link to docs on ServiceBindings and ServiceInstances here, for people who may not be familiar with them?

Reply:
+1